src/main/resources/export/cpp/RobotContainer.cpp

```diff
@@ -0,0 +1,79 @@
+#header()
+
+\#include "RobotContainer.h"
+\#include <frc2/command/button/JoystickButton.h>
+
+#@autogenerated_code("includes", "")
+#parse("${exporter_path}OI-includes.h")
+#end
+
+#set($cmd = $robot.getProperty("Autonomous Command").getValue())
+
+#foreach ($component in $components)
+#if ($helper.exportsTo("OI", $component)
+     && "#type($component)" != "" 
+     && "#type($component)" == "Joystick")
+    #set($joy = $component)
+
+#end
+#end
+
+RobotContainer::RobotContainer() : m_autonomousCommand(
+#foreach( $component in $components )
+#if($component.getBase().getType() == "Command" && !($component.getProperty("Parameters").getValue().isEmpty()))
+m_setpoint,
+#end
+#if ($component.getBase().getType() == "Command"
+     && $component.getProperty("Button on SmartDashboard").getValue())
+        #if( $component.getProperty("Parameter presets").getValue().isEmpty() )
+                #if ($component.getProperty("Requires").getValue() != "None")
+m_#required_subsystem($component))#if($component.getProperty("Add Timeout").value == true).withTimeout($component.getProperty("Timeout").value)#end{
```

Comment:
This needs a similar fix as in the robotcontainer.h for dupblicate subystems and needs to be in an autogenerated block.